tests/test.cpp

```diff
@@ -2671,6 +2681,47 @@ void CreateSharedStringTest() {
   TEST_EQ((*a[6]) < (*a[5]), true);
 }
 
+void FixedLengthArrayTest() {
+  flatbuffers::FlatBufferBuilder fbb;
+  int array[15];
+  int32_t nArr1[2] = { 1, 2 }, nArr2[2] = { 3, 4 };
+  MyGame::Example::TestEnum eEnum[2] = { TestEnum_C, TestEnum_A };
+  MyGame::Example::NestedStruct nStruct1(nArr1, TestEnum_B, eEnum);
+  MyGame::Example::NestedStruct nStruct2(nArr2, TestEnum_C, eEnum);
+  MyGame::Example::NestedStruct nStruct[2];
+
+  // Workaround for VS10 which does not support initializer lists correctly
+  std::memcpy(reinterpret_cast<void *>(&nStruct[0]),
+              reinterpret_cast<const void *>(&nStruct1), sizeof(nStruct1));
+  std::memcpy(reinterpret_cast<void *>(&nStruct[1]),
+              reinterpret_cast<const void *>(&nStruct2), sizeof(nStruct2));
+
```

Comment:
`std::vector<MyGame::Example::NestedStruct> nStruct` - ?
With `aStruct(2, array, 12, &nStruct[0]);`

Reply:
Changed to std::vector.